src/Hooks/DynamicElements.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+class DynamicElements implements \IWPML_Frontend_Action, \IWPML_DIC_Action {
+
+	public function add_hooks() {
+		add_filter( 'elementor/frontend/builder_content_data', [ $this, 'convert' ] );
+	}
+
+	/**
+	 * @param array $dataArray
+	 *
+	 * @return array
+	 */
+	public function convert( array $dataArray ) {
+		foreach ( $dataArray as &$data ) {
```

Comment:
@strategio we already know that the input is an array. Calling it `$dataArray` is quite redundant.

It seems like you called it like that because you were using `$data` to represent each item of the array.

If you change `$data` to `$item`, you can rename `$dataArray` to `$data`.

Reply:
@andreasciamanna, this is the naming we use in other places of the code. But I agree that "data" and "item" is clearer. Changed!